lib/msf/core/opt_base.rb

```diff
@@ -187,7 +201,11 @@ def display_value(value)
     # Aliases for this option for backward compatibility
     #
     attr_accessor :aliases
-
+    # 
+    # The max length of the input value
+    #
+    attr_accessor :maxLength
```

Comment:
In Ruby, object fields are named in `snake_case` like other variables, so this becomes `attr_accessor :max_length`.